test/ToastNotificationTests/APITests.cpp

```diff
@@ -323,5 +323,15 @@ namespace Test::ToastNotifications
         {
             RunTestUnpackaged(L"VerifyToastExpiresOnReboot", testWaitTime());
         }
+
+        TEST_METHOD(VerifyShowToast)
```

Comment:
VerifyShowToast_Packaged

Reply:
Similar resolution as in the other comment regarding test naming.